app/components/shared/FormYAMLEditorField/codemirror.js

```diff
@@ -1,6 +1,6 @@
 export default from 'codemirror';
 
-import 'app/css/codemirror';
+import 'app/css/codemirror.css'; // eslint-disable-line import/extensions
```

Comment:
Should we [allow CSS extensions](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/extensions.md) explicitly, rather than using `disable-line`?

Reply:
Yeah, was pondering that a bit too — I kinda feel like I like the extra terseness when importing non `*.js` files, what d’you reckon?

Reply:
Have done. 🙃🕺🏼